readme.md

```diff
@@ -164,6 +164,7 @@ Thanks for contributing! 🦋🙌
 - [](# "esc-to-deselect-line") [Adds a keyboard shortcut to deselect the current line: <kbd>esc</kbd>.](https://github.com/sindresorhus/refined-github/issues/1590)
 - [](# "html-preview-link") [Adds a link to preview HTML files.](https://user-images.githubusercontent.com/44045911/67634792-48995980-f8fb-11e9-8b6a-7b57d5b12a2f.png)
 - [](# "action-used-by-link") [Lets you see how others are using the current Action in the Marketplace.](https://user-images.githubusercontent.com/8360597/80250140-86d9c080-8673-11ea-9d28-f62faf9fd3d4.png)
+- [](# "vertical-front-matter") [Show Markdown front matter as vertical table.](https://user-images.githubusercontent.com/184316/86918308-c9655200-c126-11ea-8e10-31013667dd76.png)
```

Comment:
the screenshots (here and in `vertical-front-matter.tsx` ) are the screenshot from my issue, but your screenshot looks different

(no header, and left `th` are not have a `text-align: right`)

Reply:
You are right, I wrote the description first and forgot to update the screenshot here.